src/test/java/DukeTest.java

```diff
@@ -0,0 +1,11 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
```

Comment:
Having looking through your test function I could only find the sample test that was given. You may add more tests in specific classes. Make sure to allow access to the main/java classes that you want to refer to.

Reply:
Thank you for pointing that out! If I have time after finishing this week's project, I will try to add more tests to make it complete. Thanks!